PayPal/src/test/java/com/braintreepayments/api/paypal/PayPalClientUnitTest.java

```diff
@@ -100,7 +100,7 @@ public void createPaymentAuthRequest_callsBackPayPalResponse_sendsStartedAnalyti
 
         verify(braintreeClient).sendAnalyticsEvent(
             PayPalAnalytics.TOKENIZATION_STARTED,
-            new AnalyticsEventParams(null, null, false, null, null, null)
+            new AnalyticsEventParams(null, null, true, null, null, null)
```

Comment:
This test failed after making the change. The vault boolean defaulted to false before but the request in this test is a `PayPalVaultRequest`.